velox/docs/functions/presto/conversion.rst

```diff
@@ -496,21 +496,23 @@ By default, casting a timestamp to a string returns ISO 8601 format with an empt
 as separator between date and time.
 
 If legacy_cast configuration property is true, the result string uses character 'T'
-as separator between date and time.
+as separator between date and time and the year part is not padded.
```

Comment:
Let's describe padding behavior for the default case.

```
The year part is padded with zeros to 4 characters.
```

BTW, what happens with years that are > 4 digits? Can you add an example?

Is it possible to have negative years?

Reply:
Examples are added for years > 4 digits and negative years. Both are allowed. Negative years are also padded to 4 digits, if legacy_cast = false.